scripts/build_parser/parse_ctest_log.rb

```diff
@@ -69,6 +69,12 @@
 MAXSCALE_COMMIT_HR = "MaxScale commit"
 MAXSCALE_COMMIT_MR = "maxscale_commit"
 
+MAXSCALE_SOURCE_HR = "MaxScale source"
+MAXSCALE_SOURCE_MR = "maxscale_source"
```

Comment:
I know that currently the term *source* is used to describe the branch that is being tested. I propose to change the column name to *maxscale_branch*, so it would be more descriptive.

Reply:
Leave the name source as we discussed today on the conference.